runtime/lua/vim/lsp/health.lua

```diff
@@ -0,0 +1,25 @@
+local M = {}
+
+--- Performs a healthcheck for LSP
+function M.check_health()
+  local report_info = vim.fn['health#report_info']
+  local report_warn = vim.fn['health#report_warn']
+
+  local log = require('vim.lsp.log')
+  local current_log_level = log.get_level()
+  report_info(string.format("Current log level : %s", log.levels[current_log_level]))
+
+  if current_log_level < log.levels.WARN then
+    report_warn(string.format("Using non-default log level will result in poor runtime performance and increased disk usage"))
+  end
+
+  local log_path = vim.lsp.get_log_path()
+  report_info(string.format("Current log path: %s", log_path))
+
+  local log_size = vim.loop.fs_stat(log_path).size
+
+  report_info(string.format("Current log size: %d KB", log_size / 1000 ))
```

Comment:
Would suggest making this a warning if greater than 100MB.

```suggestion
  report_info(string.format("Log size: %d KB", log_size / 1000 ))
```